### DISCUSSION TITLE
MFG

### DISCUSSION THREAD

**Discussion by lgulyas1972** [deepmind/open_spiel#826] [Q&A] [answered]
Dear All,

First, Let me thank you for OpenSpiel. It is really a wonderful platform.

My group is interested in Mean-Field Games. I realize this is work in progress, but any advice about the status or about ways to use it or possibly, collaborate would be very welcome. Unfortunately, most people here use Python as a main language.

Best regards,
-- Laszlo

**Comment by lanctot** [comment, marked as answer]
Hi @lgulyas1972,

Python is very well-supported for MFGs. A good starting point is here: https://github.com/deepmind/open_spiel/tree/master/open_spiel/python/mfg, and specifically the examples folder in there. You likely won't need to interact with any C++ at all.

There is a sub-team of people who maintain the MFG code in OpenSpiel. I will tag them here and ask them to follow-up: @sarahperrin @perolat @sgirgin @RaphaelMarinier @PaulFMMuller @TheoCabannes.

As for the status: there are several games (search for "Mean Field" on the [games page](https://github.com/deepmind/open_spiel/blob/master/docs/games.md)) and algorithms (search for MFG on the [algorithms page](https://github.com/deepmind/open_spiel/blob/master/docs/algorithms.md)) which have already been used in papers. 

If you have any specific questions at this point, please ask here and I'm sure the team will have answers!

**Reply by lanctot** [reply]
Hi @lgulyas1972, just following up to see if you had any specific questions at this point?
